app/db/json/contacts.json

```diff
@@ -0,0 +1,50 @@
+[
+{
+"id": "e1eed2ab-0b95-40f5-ab6d-6943ffff8fc8",
+"name": "បណ្ដាញទូរស័ទ្ទជំនួយកុមារកម្ពុជា(1280)",
+"value": "1280",
+"type": "Contacts::Hotline",
+"display_order": 1,
+"updated_at": "2023-08-03T11:10:06.500+07:00",
+"contact_directory": {
+"id": "e63469e6-7939-4efa-be75-375bf1d5517d",
+"name": "សេវាគាំទ្រផ្លូវចិត្ត"
+}
+},
+{
+"id": "fa2f7afb-adce-45ac-a4b2-c72463735d3a",
+"name": "បណ្ដាញទូរស័ទ្ទជំនួយកុមារកម្ពុជា(1280)",
+"value": "1280",
+"type": "Contacts::Sms",
+"display_order": 2,
+"updated_at": "2023-08-03T11:10:06.509+07:00",
+"contact_directory": {
+"id": "e63469e6-7939-4efa-be75-375bf1d5517d",
+"name": "សេវាគាំទ្រផ្លូវចិត្ត"
+}
+},
+{
+"id": "dc7a8e24-fd3e-4fe7-a5cf-bf235d3b061c",
+"name": "បណ្ដាញទូរស័ទ្ទជំនួយកុមារកម្ពុជា(1280)",
+"value": "http://m.me/chc1280",
+"type": "Contacts::Messenger",
+"display_order": 3,
+"updated_at": "2023-08-03T11:10:06.518+07:00",
+"contact_directory": {
```

Comment:
@limkimsan please kindly update to store only contact_directory_uuid but not hash({})